### ISSUE TITLE
[Accessibility] Contrast issues with text in Additional checks

### ISSUE THREAD

**Issue by PeterSaathoff-Harshfield** [ampproject/amp.dev#4506]
Environment:
Any OS/browser

Steps to reproduce:
1. Open https://amp-dev-staging.appspot.com/page-experience/
2. Enter a valid URL and press Analyze

Under the "Additional checks" heading, observe the text

"Passed"
Text color: #00CD3C
Contrast ratio: 2.14:1

"Check manually"
Text color: #CCCCCC
Contrast ratio: 1.61:1

The color contrast ratio should be at least 4.5:1. See [WCAG 1.4.3](https://www.w3.org/TR/WCAG21/#contrast-minimum) and [Understanding Success Criterion 1.4.3](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html)

If there are alternate views that use yellow #F39C12, that color also needs to be updated.

![image](https://user-images.githubusercontent.com/30604285/92977527-1c6ac680-f442-11ea-90ea-28df83ef7c37.png)
